packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -131,8 +132,9 @@ export class TabBarRenderer extends TabBar.Renderer {
     /**
      * If size information is available for the label, set it as inline style. Tab padding
      * and icon size are also considered in the `top` position.
+     * @param {boolean} [isInSidePanel] - Whether the label is in the side panel.
```

Comment:
I think you can skip the typing/brackets part, since it is already defined in the implementation? Documentation is good, just trying to avoid being redundant :)